src/components/sidebar/sidebar.component.tsx

```diff
@@ -20,6 +21,8 @@ interface SidebarProps {
   selectedPage: string;
   setValues: (values: unknown) => void;
   values: object;
+  collapsedPages: Set<string>;
+  togglePageCollapse: (pageId: string) => void;
```

Comment:
Did u add some UI elements to update the page state?

Reply:
For context, that component renders this section of the form engine. It was previously removed during the RHF migration. Notice how it shows only the section title and it's sole purpose to quickly scroll to a specific section of the form. That's what i mean't when i asked this. 

<img width="1029" alt="Screenshot 2024-09-19 at 14 15 31" src="https://github.com/user-attachments/assets/eb3d6e57-568a-40ef-9349-ddde33313ddd">

Reply:
Oh okay, i was under the wrong impression. Since that is the case there is no need to make changes to the sidebar component or styling